CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.0.0
 
 - BREAKING CHANGE: Fixed context screenDpi is of type int #58
+- BREAKING CHANGE: Renamed capture method to captureEvent #64
```

Comment:
This will go to 4.0.0 if #65 gets merged

Reply:
SGTM